### DISCUSSION TITLE
Newbie question: windows solution

### DISCUSSION THREAD

**Discussion by xoxolo** [ultralytics/yolov5#7219] [Q&A]
Good day,
I did try it on my station and it's working beautifully, thanks a lot for this.
Now, I want to unpack it and create an .EXE file to install and run it on windows.
I'm quite lost about the next step, can you provide me some help?
Thanks bunches!
![bitmoji](https://sdk.bitmoji.com/render/panel/732eaa6e-2db0-48dc-b8d8-4484ec74e70f-55a93d13-467d-44b2-9732-fd6a41089c3a-v1.png?transparent=1&palette=1&width=246)

**Comment by glenn-jocher** [comment]
@xoxolo the scope of our support is bugs and feature requests in YOLOv5, but if you are much more specific perhaps a community member might be able to provide you insight into your request.

**Reply by xoxolo** [reply]
Thx Glenn-jocher!
I want to take it out with my custom training dataset, then package it to run it on windows 10 on another computer.
That package will have to trigger the installation of all the requirements in order to make it happen.

**Reply by xoxolo** [reply]
as an addon, and then, you will get how newbie I am, you did deployed it as an app for Iphone for example, I want tot do the same for windows, hope that helps.

**Reply by glenn-jocher** [reply]
@xoxolo Docker Image is a standalone environment that comes with all dependencies installed. 

## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab and Kaggle** notebooks with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://github.com/ultralytics/yolov5/wiki/GCP-Quickstart)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://github.com/ultralytics/yolov5/wiki/AWS-Quickstart)
- **Docker Image**. See [Docker Quickstart Guide](https://github.com/ultralytics/yolov5/wiki/Docker-Quickstart) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

**Reply by xoxolo** [reply]
thanks a lot!
